packages/xo-server-sdn-controller/src/index.js

```diff
@@ -1177,6 +1177,15 @@ class SDNController extends EventEmitter {
     }
 
     const tunnel = this._getHostTunnelForNetwork(host, network.$ref)
+    if (tunnel === undefined) {
+      log.info('Unable to add host to network: no tunnel available', {
+        network: network.name_label,
+        host: host.name_label,
+        pool: host.$pool.name_label,
```

Comment:
As I said before, name labels are not reliable (not unique) to do proper debugging.

Maybe fix this in another PR.

Reply:
Yes it's on my TODO list. :)